### DISCUSSION TITLE
cmsadmin page build just show development group

### DISCUSSION THREAD

**Discussion by fanlt** [luyadev/luya#2141] [Questions & Problems] [answered]
![image](https://user-images.githubusercontent.com/13359458/169507511-9dd8536b-3733-4d9a-8216-c619c28a9049.png)
Just show development group ,I can not search list etc..

**Comment by nadar** [comment, marked as answer]
after running the import command, blocks should be available. Depending on the modules you have installed. But based on your topic https://github.com/luyadev/luya/discussions/2140 it seems not to install the blocks.

**Reply by nadar** [reply]
Might have been related to: https://github.com/orgs/luyadev/discussions/2140
